openshift-knative-operator/hack/update-manifests.sh

```diff
@@ -12,42 +12,53 @@ source "$root/hack/lib/__sources__.bash"
 # be fairly stable.
 serving_files=(serving-crds serving-core serving-hpa serving-domainmapping-crds serving-domainmapping serving-post-install-jobs)
 eventing_files=(eventing-crds eventing-core in-memory-channel mt-channel-broker eventing-sugar-controller)
+kafka_files=(channel-consolidated source)
 
 function download {
   component=$1
   version=$2
+  org=$3
+  shift
   shift
   shift
 
   files=("$@")
+  local component_dir=""
+  local target_dir=""
+  if [[ $component == "eventing-kafka" ]]; then
+      component_dir="$root/knative-operator/deploy/resources/knativekafka/"
+      target_dir="${component_dir}/"
+  else
+      component_dir="$root/openshift-knative-operator/cmd/operator/kodata/knative-${component}"
+      target_dir="${component_dir}/${version:1}"
+      rm -r "$component_dir"
+      mkdir -p "$target_dir"
+  fi
 
-  component_dir="$root/openshift-knative-operator/cmd/operator/kodata/knative-${component}"
-  target_dir="${component_dir}/${version:1}"
-  rm -r "$component_dir"
-  mkdir -p "$target_dir"
-  
   for (( i=0; i<${#files[@]}; i++ ));
   do
     index=$(( i+1 ))
     file="${files[$i]}.yaml"
     target_file="$target_dir/$index-$file"
-    url="https://github.com/knative/$component/releases/download/$version/$file"
+    url="https://github.com/$org/$component/releases/download/$version/$file"
 
     wget --no-check-certificate "$url" -O "$target_file"
   done
 }
 
-download serving "$KNATIVE_SERVING_VERSION" "${serving_files[@]}"
+download serving "$KNATIVE_SERVING_VERSION" "knative" "${serving_files[@]}"
```

Comment:
I think we should also look at this file:
https://github.com/openshift-knative/serverless-operator/blob/main/olm-catalog/serverless-operator/project.yaml#L26

and add there the pointer to our own 0.20.0 branch:
https://github.com/openshift-knative/eventing-kafka/tree/release-v0.20.0